app/src/main/kotlin/viewmodel/graph/GraphViewModel.kt

```diff
@@ -64,9 +65,8 @@ class GraphViewModel<D>(
         val edge = graph.addEdge(firstVertexVM.vertex, secondVertexVM.vertex)
 
         _edgeViewModels = _edgeViewModels.toMutableMap().apply {
-            this[edge] = EdgeViewModel(firstVertexVM, secondVertexVM, edge)
+            this[edge] = EdgeViewModel(firstVertexVM, secondVertexVM, isDirected)
         }
-        TestRepresentation().place(740.0, 650.0, verticesVM)
```

Comment:
Is this necessary?

Reply:
I think its quite logic, to update representation after add

Reply:
I don't remember exactly what was the reason I deleted it, but it works without it just fine